package.json

```diff
@@ -1,12 +1,13 @@
 {
-  "name": "react-chatbot-kit",
-  "version": "2.2.0",
+  "name": "react-chatbot-kit-for-gpt4-vision",
```

Comment:
Lets keep the same name as the repo name `react-chatbot-kit-extension`.
With your current name if gpt5 arrive we have to change it.

Reply:
resolved